CSS/utility.css

```diff
@@ -133,14 +147,19 @@
   max-height: 9.375rem;
   overflow: scroll;
   overflow-x: hidden;
+  /* overflow-x: visible;
+  max-width: 60rem; */
 }
 
 .background-cover {
   background-color: var(--background-cover);
   border: 1px solid var(--secondary-color);
 }
 
-
 .round {
   border-radius: 50%;
+}
+
+.width-300 {
```

Comment:
Can have better class name. What is 300 specifying here?  Let me know your thoughts on this.

Reply:
300 means width of 300px, I did not get a better name for it